app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -3808,6 +3919,24 @@ class ChatActivity :
         startActivity(shareIntent)
     }
 
+    fun joinOneToOneConversation(userId: String) {
+        val apiVersion =
+            ApiUtils.getConversationApiVersion(conversationUser!!, intArrayOf(ApiUtils.API_V4, 1))
+        val retrofitBucket = ApiUtils.getRetrofitBucketForCreateRoom(
+            apiVersion,
+            conversationUser?.baseUrl!!,
+            "1",
+            "users",
```

Comment:
can you add constants for
"1"
and
"users"

and also use them in the other places in this class where
getRetrofitBucketForCreateRoom is used?

(better would be to have these types centralized somewhere but doesn't have to be done in this PR)